### ISSUE TITLE
🐛 Bug: Shared view link access (in incognito) redirects to sign-in page

### ISSUE THREAD

**Issue by dstala** [nocodb/nocodb#4694]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Observed on `develop` branch; works fine in `master`

- create shared view
- access link in incognito mode
- redirects to sign in page

```
authMiddleware :: Unauthorized [Error]: Unauthorized access
    at Function.unauthorized (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/meta/helpers/catchError.ts:44:11)
    at authMiddleware (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/meta/helpers/ncMetaAclMw.ts:38:17)
    at /Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/meta/helpers/catchError.ts:6:20
    at Layer.handle [as handle_request] (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/node_modules/express/lib/router/route.js:144:13)
    at exports.default (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/meta/helpers/extractProjectIdAndAuthenticate.ts:144:5)
```

### Desired Behavior

access should work without redirection to sign-in page

### Project Details

- In develop

**Comment by fortrieb** [comment]
Hi,

have installed Version 0.101.0 and these bug is still there.

### Steps to reproduce

1. Create a Form view
2. Share View with password protection
3. Open Share View Link in another browser/session
4. Enter correct password
5. Redirect to login page

Can anybody confirm this bug still exist?

**Comment by dib0** [comment]
I'm on 0.101.0 at the moment. In that version the issue still exists. Also, without a password the user will be redirected to the login page.

**Comment by pranavxc** [comment]
@dib0 Use our latest release which is version `0.101.2`

**Comment by fortrieb** [comment]
Issue is fixed in latest version `0.101.2`. Thanks

**Comment by dib0** [comment]
Thank you. Just updated and @fortrieb in version 0.101.2 it is fixed.
